modules/lib/lib-portal/src/main/resources/lib/xp/examples/portal/apiUrl.js

```diff
@@ -8,14 +8,22 @@ const url = portalLib.apiUrl({
     params: {
         a: 1,
         b: 2
-    }
+    },
+    path: 'mypath'
```

Comment:
`mypath/subpath` would be a better example

Reply:
I don't agree with that. You left the comment https://github.com/enonic/xp/pull/10698#discussion_r1771623612 where your idea is better, than you wrote here.